internal/helper/helper_test.go

```diff
@@ -22,102 +22,99 @@ func mockRename(string, string) error {
 
 // TestRestoreResolvConf tests if resolv.conf is restored correctly
 func TestRestoreResolvConf(t *testing.T) {
-	t.Run("test_restore_resolv_conf", func(t *testing.T) {
-		asserter := Asserter{T: t}
-		// Prepare temporary directory
-		workDir := filepath.Join("/tmp", "ubuntu-image-"+uuid.NewString())
-		err := os.Mkdir(workDir, 0755)
-		asserter.AssertErrNil(err, true)
-		defer os.RemoveAll(workDir)
+	t.Parallel()
```

Comment:
This is a nice feature! From what I recall, the Python regular unittest module couldn't run tests in parallel, for instance. So that's a win.

Reply:
In fact by default go will run tests in parallel. But tests from different packages. This option is used to tell that in the same package, some tests can be run in parallel with others of the same package. This is nice in our case because we do not have a lot of packages and most of our tests are gathered in a single one (statemachine).